packages/test/test-gc-sweep-tests/src/test/gcSweepTest.spec.ts

```diff
@@ -74,12 +75,14 @@ describeNoCompat("GC Sweep tests", (getTestObjectProvider) => {
         runtimeOptions,
     );
 
-    const sessionExpiryDurationMs = 3000; // 3 seconds
+    // const sessionExpiryDurationMs = 3000; // 3 seconds
+    const inactiveTimeoutMs = 3000; // 3 seconds
 
     // Set settings here, may be useful to put everything in the mockConfigProvider
     const settings = {
-        "Fluid.GarbageCollection.RunSessionExpiry": "true",
-        "Fluid.GarbageCollection.TestOverride.SessionExpiryMs": sessionExpiryDurationMs,
+        // "Fluid.GarbageCollection.RunSessionExpiry": "true",
+        "Fluid.GarbageCollection.TestOverride.InactiveTimeoutMs": inactiveTimeoutMs,
```

Comment:
After enabling this, I noticed that my tests would only run once. After removing it, the test would run 10 times.

Reply:
This sounds like a bug, maybe open a work item in ADO?

Reply:
AB#1844 created